HACKING.adoc

```diff
@@ -81,3 +81,24 @@ See the `tests/tests.sh` file to see how this works, and if you want to check ad
 
 The backend services are written in JavaScript and have tests written primarily
 in link:https://mochajs.org/[Mocha], which can be run via `make check`.
+
+=== How to list available plugins updates
+
+Given the /pluginManager is made unreachable on purpose, we cannot use the standard UI easily to check what plugins could receive an update.
```

Comment:
This is not necessary, `make propose-updates` will work just fine :smile: 

I'll rework this section of documentation